pagopa/io-app
# fix: [IOBP-879,IOBP-880] Upgrade `react-native-vision-camera` to version `4.3.1`

## Short description
This PR upgrades the `react-native-vision-camera` version to the v.`4.3.1` that fixes two of the bugs that are also reported in their issues.
1. [IOBP-879] The Camera is working fine on most devices. However, there are a few on which it just won't work. The camera will simply try to load and stay black. (https://github.com/mrousavy/react-native-vision-camera/issues/2716)
2. [IOBP-880] On some devices the camera image appears horizontally stretched (https://github.com/mrousavy/react-native-vision-camera/issues/2364)

## How to test
- Please test this PR on a real device to check that the camera is working correctly from the Scan section tab bar.


[IOBP-879]: https://pagopa.atlassian.net/browse/IOBP-879?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IOBP-880]: https://pagopa.atlassian.net/browse/IOBP-880?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
